wagtail/admin/templates/wagtailadmin/shared/dialog/dialog.html

```diff
@@ -0,0 +1,36 @@
+{% load wagtailadmin_tags i18n %}
```

Comment:
it is worth setting up a template var or 'with' for the id, so save having to write it multiple times

`{% with id=title|cautious_slugify %} `

or a template tag?

`{% get_id from=title as id %}`

Reply:
Great idea! I've went with the `{% with %}` option